### PR TITLE
fix(implementsPlugin): check if reflection has children

### PR DESCRIPTION
**Pull request by jiayihu**
Fixes https://github.com/TypeStrong/typedoc/issues/377

### CODE REVIEW

File: src/lib/converter/plugins/ImplementsPlugin.ts [hunk 1]
```diff
@@ -38,6 +38,9 @@ export class ImplementsPlugin extends ConverterComponent
             }
 
             var classMember:DeclarationReflection;
+
+            if (!classReflection.children) return;
```
Comment (aciccarello) [inline src/lib/converter/plugins/ImplementsPlugin.ts]: This should do the trick. The style does seem inconsistent though. I think the return should either be on a new line (like the `if (!classMember)`), or wrap the for loop.

**Comment by jiayihu** [comment]
@aciccarello fixed style.

**Comment by aciccarello** [comment]
Thanks for adjusting that. I know it's small but as someone who doesn't know all the code the consistency helps.

**Comment by jiayihu** [comment]
No problem, I always use linters in my projects so I perfectly understand 👍

**Pull request merged**